addons/udes_stock/models/stock_move.py

```diff
@@ -143,52 +143,97 @@ def split_out_move_lines(self, move_lines, **kwargs):
 
         return new_move
 
-    def split_out_incomplete_move(self, **kwargs):
+    def get_uncovered_moves(self, mls):
+        """Return the moves in self not covered by mls regardless of the mls state"""
```

Comment:
It would be helpful if this docstring defined "covered".

Reply:
Agreed, I'm not sure I understand what the function does just by looking at the docstring or name.